scripts/SANS/sans/gui_logic/gui_common.py

```diff
@@ -4,11 +4,14 @@
 #     NScD Oak Ridge National Laboratory, European Spallation Source
 #     & Institut Laue - Langevin
 # SPDX - License - Identifier: GPL - 3.0 +
-from sans.common.enums import SANSInstrument, ISISReductionMode, DetectorType
-from qtpy.QtWidgets import (QFileDialog)  # noqa
-from qtpy.QtCore import (QSettings)  # noqa
 import os
 
+from qtpy.QtCore import (QSettings)  # noqa
+from qtpy.QtWidgets import (QFileDialog)  # noqa
```

Comment:
Do we need the brackets and `noqa` on these imports?

Reply:
I don't think so, I don't know why I added them